students/alexander_boone/lesson04/assignments/mailroom.py

```diff
@@ -29,85 +30,118 @@ def thank_you():
         # run if user inputs 'list'
         if name == 'list':
             list_names = list()
-            for item in donor_data:
+            for item in donors:
                 list_names.append(item['first_name'] + ' ' + item['last_name'])
             print(list_names)
         # run if name that user inputs is not already in the donor list
-        elif not any(donor['first_name'] == name.split(' ')[0] and donor['last_name'] == name.split(' ')[1] for donor in donor_data):
+        elif not any(donor['first_name'] == name.split(' ')[0] and
+                     donor['last_name'] == name.split(' ')[1] for
+                     donor in donors):
             firstName = name.split(' ')[0]
             lastName = name.split(' ')[1]
-            donor_data.append({'first_name': firstName,'last_name':lastName, 'donation_total': 0, 'donation_count': 0})
-            index = len(donor_data) - 1
+            donors.append({'first_name': firstName, 'last_name': lastName,
+                           'donation_total': 0, 'donation_count': 0})
+            index = len(donors) - 1
         # run if name that user inputs is already in the donor list
         else:
             firstName = name.split(' ')[0]
             lastName = name.split(' ')[1]
-            for i in range(len(donor_data)):
-                if donor_data[i]['first_name'] == firstName and donor_data[i]['last_name'] == lastName:
+            for i in range(len(donors)):
+                if (donors[i]['first_name'] == firstName and
```

Comment:
please conform to pep8 style guidelines, var names (and method names) should be snake_case style -> `first_name`